exceptions_tests.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from pip._vendor.distlib.compat import raw_input
+
+def csv_data_is_supplied_test():
+   try:
+       csv_path_from_user = raw_input("Please insert the csv path: ")
+       test = pd.read_csv(csv_path_from_user)
```

Comment:
read_csv and raw_input can't be part of this function. pls seperete them to diffrent functions...

Reply:
done :)